addons/account/models/account_move.py

```diff
@@ -3233,10 +3233,13 @@ def add_file_data_results(file_data, invoice):
 
                 except RedirectWarning:
                     raise
-                except Exception:
+                except Exception as e:
                     message = _("Error importing attachment '%s' as invoice (decoder=%s)", file_data['filename'], decoder.__name__)
                     current_invoice.sudo().message_post(body=message)
                     _logger.exception(message)
+                    if isinstance(e, UserError):
+                        message = _("An error occurred during the import:") + Markup("<br/><br/>%s") % str(e)
+                        current_invoice.sudo().message_post(body=message)
```

Comment:
Do we really need to have two messages in the chatter in case of an UserError? If not, maybe the message could be combined in those cases.

```suggestion
                    _logger.exception(message)
                    if isinstance(e, UserError):
                        message += Markup("<br/><br/>%s") % str(e)
                    current_invoice.sudo().message_post(body=message)
```

Reply:
I find it less user friendly, no? maybe @pmax-x could give us his opinion on this?

Reply:
In my opinion two messages could lead to confusion, leading to think two errors took place. But if you prefer to leave it as it is, that's ok, I'll just pass it on to the final reviewer :)